### PR TITLE
Divide equation in two parts

### PR DESCRIPTION
**Pull request by manuel489**
*As requested in https://github.com/metanorma/metanorma-bipm/issues/35#issuecomment-715528653*

Thanks!

### CODE REVIEW

**Comment by ronaldtse** [comment]
@manuel489 do we need align-left?

File: sources/sections-en/04-non-si-units.adoc [hunk 1]
```diff
@@ -38,7 +38,9 @@ Nonetheless it is recognized that some non-SI units are widely used and are expe
 ====
 The gal (symbol: stem:["Gal"]) is a non SI unit of acceleration employed in geodesy and geophysics to express acceleration due to gravity
 
-stem:[1 " Gal" = 1 " cm s"^(−2) = 10^(−2) " m s"^(−2)]
+[align=left]
```
Comment (ronaldtse) [inline sources/sections-en/04-non-si-units.adoc:41]: Align left needed? Feels a bit strange...?

**Review by ronaldtse — APPROVED** [review]

File: sources/sections-en/04-non-si-units.adoc (hunk 1 above)
Comment (manuel489) [reply sources/sections-en/04-non-si-units.adoc:41]: Apparently, it is needed for Word output according to the [documentation](https://www.metanorma.com/author/topics/document-format/text/), that says:

*If you have line-breaks in a paragraph, and the default alignment in the stylesheet is justified (as is often the case in Word output), you will need to set `[align=left]` to make the paragraph look as normally expected*

Comment (ronaldtse) [reply sources/sections-en/04-non-si-units.adoc:41]: I see! This document is only generated via PDF, so perhaps we don't need it here. Thanks.

Comment (manuel489) [reply sources/sections-en/04-non-si-units.adoc:41]: But maybe in this case it doesn't apply? @opoudjis?

Comment (manuel489) [reply sources/sections-en/04-non-si-units.adoc]: > I see! This document is only generated via PDF, so perhaps we don't need it here. Thanks.

Oh, I hadn't read this. Ok, changes applied!

**Pull request merged**